stable/yugabyte/templates/service.yaml

```diff
@@ -246,6 +246,7 @@ spec:
         {{ else }}
 {{ toYaml $root.Values.resource.tserver | indent 10 }}
         {{ end }}
+        workingDir: "/mnt/disk0"
```

Comment:
Add a comment linking to the issue number and explaining why we did this change.